dataflow/src/main/java/org/checkerframework/dataflow/cfg/builder/CFGTranslationPhaseOne.java

```diff
@@ -1805,7 +1804,15 @@ public Node visitCompoundAssignment(CompoundAssignmentTree tree, Void p) {
                         assert (kind == Tree.Kind.PLUS_ASSIGNMENT);
                         Node targetRHS = stringConversion(targetLHS);
                         value = stringConversion(value);
-                        Node r = new StringConcatenateAssignmentNode(tree, targetRHS, value);
+                        BinaryTree operTree =
```

Comment:
Looking at other `operTree`s in this method, it seems we need to call `handleArtificialTree(operTree);`

Reply:
Yes.

Reply:
There is an override that does something and as these are new artificial trees, you should call the method.

Reply:
Yeah, fixed.